### ISSUE TITLE
Missing v3.11 version for armhf and armv7 architectures

### ISSUE THREAD

**Issue by tmaczukin** [multiarch/alpine#30]
The latest CI jobs for [armhf](https://travis-ci.org/multiarch/alpine/jobs/643300507) and [armv7](https://travis-ci.org/multiarch/alpine/jobs/643300510) architectures have failed, which makes the v3.11 images for these unavailable.

All other jobs for all other version/architecture pairs are passing, which suggests that the problem was temporary.

Could the jobs be restarted and the missing version tags published?

**Comment by wborn** [comment]
It seems to be failing ever since it was added in https://github.com/multiarch/alpine/pull/28. :-/

**Comment by alejandro-perez** [comment]
I'm also interested in this issue. Is there any chance this will be fixed soon?

Many thanks for the hard workd!

**Comment by alejandro-perez** [comment]
In the meanwhile, if you really need a armhf version of 3.11 you can do the following to build your own image:
```
FROM multiarch/alpine:armhf-v3.10
RUN sed -i "s/v3.10/v3.11/" /etc/apk/repositories \
         && apk update && apk upgrade
```

**Comment by wilmardo** [comment]
I looked into this when I opened the PR and the issue doesn't seem temporary.
I tried restarting it for the PR to no avail. Since the CentOS image also has issues and QEMU segfaults I concluded there is an issue with QEMU. Therefore tried updating to the latest but that hasn't made a difference so far.

I will give debugging this another try now :)

**Comment by lafin** [comment]
Try using the latest stable version 3.12. It seems that the Alpine developers broke version 3.11 for arm, I expected them to fix version 3.11, but this did not happen.
